aikau/src/test/resources/alfresco/services/SearchServiceTest.js

```diff
@@ -84,6 +84,87 @@ define(["intern!object",
             });
       },
 
+      "Repository scope should be used by default": function() {
+         return browser.findByCssSelector("body").end().getLastPublish("ALF_SEARCH_REQUEST")
+            .then(function(payload) {
+               assert.propertyVal(payload, "repo", true, "Repository scope not requested");
+            })
+         .end()
+         .findByCssSelector("tr.mx-row:nth-child(1) td.mx-url")
+            .getVisibleText()
+            .then(function(url) {
+               assert.include(url, "repo=true", "Repository scope not used in XHR");
+            })
+         .end()
+         .clearLog();
+      },
+
+      "Switch to All Sites scope": function() {
+         return browser.findById("FCTSRCH_SCOPE_SELECTION_MENU_text")
+            .click()
+         .end()
+         .findById("FCTSRCH_SET_ALL_SITES_SCOPE_text")
+            .click()
+         .end()
+         .getLastPublish("ALF_SEARCH_REQUEST")
+            .then(function(payload) {
+               assert.propertyVal(payload, "repo", false, "Repository scope used");
+               assert.propertyVal(payload, "site", "", "Site incorrect");
+            })
+         .end()
```

Comment:
`end()` not necessary here as getLastPublish (and similar methods) do not change the CSS scope